examples/traces/exporters/otlp_http_from_factory.php

```diff
@@ -11,6 +11,7 @@
  */
 putenv('OTEL_EXPORTER_OTLP_ENDPOINT=http://collector:4318');
 putenv('OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf');
+putenv('OTEL_EXPORTER_OTLP_TIMEOUT=1500'); //1.5s
```

Comment:
Is 1.5s an arbitrary measurement?  Seems just as good as any; was just curious.

Reply:
This is just an example to show setting it to a non-default value does work. The official default in spec is 10s.